database/postgres/build_list_test.go

```diff
@@ -396,27 +399,31 @@ func TestPostgres_Client_GetRepoBuildList(t *testing.T) {
 	defer func() { _sql, _ := _database.Postgres.DB(); _sql.Close() }()
 
 	// create expected return in mock
-	_rows := sqlmock.NewRows([]string{"count"}).AddRow(2)
+	_rows := sqlmock.NewRows([]string{"count"}).AddRow(3)
 
 	// ensure the mock expects the query
 	_mock.ExpectQuery(`SELECT count(*) FROM "builds" WHERE repo_id = $1`).WillReturnRows(_rows)
 
 	// create expected return in mock
 	_rows = sqlmock.NewRows(
 		[]string{"id", "repo_id", "number", "parent", "event", "status", "error", "enqueued", "created", "started", "finished", "deploy", "deploy_payload", "clone", "source", "title", "message", "commit", "sender", "author", "email", "link", "branch", "ref", "base_ref", "head_ref", "host", "runtime", "distribution", "timestamp"},
-	).AddRow(1, 1, 1, 0, "", "", "", 0, 0, 0, 0, "", nil, "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", 0).
-		AddRow(2, 1, 2, 0, "", "", "", 0, 0, 0, 0, "", nil, "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", 0)
+	).AddRow(1, 1, 1, 0, "", "", "", 0, 0, 1, 0, "", nil, "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", 0).
+		AddRow(2, 1, 2, 0, "", "", "", 0, 0, 2, 0, "", nil, "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", 0)
```

Comment:
nothing to do here right now, but a comment...this many params is hard to understand and prone to errors. named arguments would be better, or a type passed that contains the parameters. something to think about for future development.

Reply:
Named params isn't quite a thing in Go, you can pass a struct or call a function on a struct to access it's fields but in this particular instance we are handicapped by the [library](https://pkg.go.dev/github.com/DATA-DOG/go-sqlmock). If you look up the [NewRow in godoc](https://pkg.go.dev/github.com/DATA-DOG/go-sqlmock#NewRows) you'll see it takes that array of strings to define what the columns are and then AddRow takes an array of a custom type which narrows down it [down to primitives](https://pkg.go.dev/database/sql/driver#Value).